### ISSUE TITLE
Better summary report notification

### ISSUE THREAD

**Issue by mani76** [containrrr/watchtower#1469]
### Is your feature request related to a problem? Please describe.

Currently notification report sends following info when checking is finished:

Updated containers
Skipped containers
Fresh containers
Failed containers

When running using monitor only mode, there is no way to know if any updates are found, because it only provides list of fresh containers.

### Describe the solution you'd like

Monitor only mode should include new report info list which contains containers which could be updated.
For example:

Updateable containers

Same info could also be added to the text message.

### Describe alternatives you've considered

Use existing Updated containers list also when not actually doing update.

**Comment by piksel** [comment]
Yeah, good point. Normally (when not using `monitor-only`) a container should always end up in either failed or updated when there is a newer version available. So that's why the reports doesn't have the concept of a `stale` (containers that have an update available) status.

**Comment by piksel** [comment]
Actually, the support is there, it's just the template that doesn't consider the stale containers. Using this template should fix it:
```
{{- if .Report -}}
  {{- with .Report -}}
    {{- if ( or .Updated .Failed .Stale ) -}}
{{len .Scanned}} Scanned, {{len .Updated}} Updated, {{len .Stale}} Stale, {{len .Failed}} Failed
      {{- range .Updated}}
- {{.Name}} ({{.ImageName}}): {{.CurrentImageID.ShortID}} updated to {{.LatestImageID.ShortID}}
      {{- end -}}
      {{- range .Stale}}
- {{.Name}} ({{.ImageName}}): {{.CurrentImageID.ShortID}} has newer image available {{.LatestImageID.ShortID}}
      {{- end -}}
      {{- range .Fresh}}
- {{.Name}} ({{.ImageName}}): {{.State}}
	  {{- end -}}
	  {{- range .Skipped}}
- {{.Name}} ({{.ImageName}}): {{.State}}: {{.Error}}
	  {{- end -}}
	  {{- range .Failed}}
- {{.Name}} ({{.ImageName}}): {{.State}}: {{.Error}}
	  {{- end -}}
    {{- end -}}
  {{- end -}}
{{- else -}}
  {{range .Entries -}}{{.Message}}{{"\n"}}{{- end -}}
{{- end -}}
```

**Comment by mani76** [comment]
Seems to work, thanks
